app/src/main/java/com/ezanetta/altoquepr/MainActivity.kt

```diff
@@ -2,10 +2,19 @@ package com.ezanetta.altoquepr
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        showEvenNumbers()
+    }
+
+    private fun showEvenNumbers() {
+        Log.d(
+            "Numbers",
+            "Even numbers: ${filterEvenNumbers(listOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10))}"
```

Comment:
Here, we could also use a [range](https://kotlinlang.org/docs/reference/ranges.html) and then convert it into a list
```suggestion
            "Even numbers: ${filterEvenNumbers((1..10).asIterable().toList())}"
```